### PR TITLE
chore: format acceptance tests

### PR DESCRIPTION
**Pull request by travissarles**

### CODE REVIEW

**Comment by jachro** [comment]
Could we maybe add this few files as a single commit to the PR you're working on?

**Comment by travissarles** [comment]
> Could we maybe add this few files as a single commit to the PR you're working on?

We could but we'll have some irrelevant formatting changes mixed with code changes. Is that okay given that there are only five files affected?

**Comment by jachro** [comment]
Actually, you're right. Why not create a new PR. I believe these are all the formatting changes in the project. Did you maybe run the scalafmt from sbt?

**Comment by travissarles** [comment]
> Actually, you're right. Why not create a new PR. I believe these are all the formatting changes in the project. Did you maybe run the scalafmt from sbt?

Yes, it seems to agree with the one from IntelliJ.

**Comment by jachro** [comment]
I was thinking more about running
```
sbt scalafmt
sbt test:scalafmt
```

**Comment by travissarles** [comment]
> I was thinking more about running
> 
> ```
> sbt scalafmt
> sbt test:scalafmt
> ```

After doing a `test:scalafmt` I noticed more formatting changes

**Comment by rokroskar** [comment]
If you want to commit to a formatting style, there should be a format check in the CI pipeline.

**Comment by travissarles** [comment]
> If you want to commit to a formatting style, there should be a format check in the CI pipeline.

Agreed. We have this for renku-graph.

File: .github/workflows/check-acceptance-test-fmt.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  formatting-check:
+    name: Scala formatting check
+    runs-on: ubuntu-latest
+    steps:
+      - name: Switch to acceptance tests directory
+        run: |
+          cd acceptance-tests
+      - name: run scalafmt
+        run: |
+          sbt test:scalafmt
```
Comment (jachro) [inline .github/workflows/check-acceptance-test-fmt.yml:15]: I guess if the step suppose to be checking the formatting we should probably use `sbt scalafmtCheck` as it fails command. `sbt test:scalafmt` simply reformats the code and never fails.

**Comment by jachro** [comment]
Maybe one more thing. Once we're working with the code formatting maybe we should bump the version of the scalafmt plugin as we've got it on renku-graph:
```
addSbtPlugin("org.scalameta" % "sbt-scalafmt" % "2.4.2")
```
And in the `scalafmt.conf`
```
version = "2.7.1"
```

**Review by jachro — APPROVED** [review]
👍

**Pull request merged**